### DISCUSSION TITLE
Use the OpenGraph image instead of the current social preview image

### DISCUSSION THREAD

**Discussion by ouuan** [cpeditor/cpeditor#812] [Ideas]
GitHub now generates OpenGraph images for public repositories, issues, commits, and pull requests when a URL is shared on platforms that support OpenGraph rendering: https://github.blog/changelog/2021-04-21-opengraph-images-for-github-repositories-commits-issues-and-pull-requests/

I think it looks great, so I'm wondering whether we should remove the [Social preview](https://github.com/cpeditor/cpeditor/settings) image to use the OpenGraph image in social media. One downside is that we'll have no image on pages such as https://github.com/topics/competitive-programming.

@coder3101 What do you think?

The current social preview image:

![](https://repository-images.githubusercontent.com/221268096/2bd0cb80-7277-11ea-9126-52eeb70263d8)

An example of an OpenGraph image:

![](https://i2.wp.com/user-images.githubusercontent.com/7900087/115299484-b3ec7200-a113-11eb-92e5-1a383fe64af0.png?ssl=1)

**Comment by coder3101** [comment]
>  > I think it looks great, so I'm wondering whether we should remove the Social preview image to use the OpenGraph image in social media. One downside is that we'll have no image on pages such as https://github.com/topics/competitive-programming.
> 
 Go ahead, it looks great. Its dynamic and shares far more information than a static image.
